eng/Version.Details.xml

```diff
@@ -4,6 +4,7 @@
     <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="6.0.0-beta.21227.1">
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>cca78ffe3eefdc217e43c2421f2f23355f16da2d</Sha>
+      <SourceBuild RepoName="arcade" ManagedOnly="true"/>
```

Comment:
We will be adorning more dependencies with the `SourceBuild` attribute but not all upstream dependencies have their SourceBuild intermediate NuGet packages created yet or msbuild is not on a version that is source built yet.  Once all repos are on ArPow we will make the appropriate updates here to eliminate prebuilts.  We will also be turning on prebuilt detection in the source build CI leg to prevent regressions.

Reply:
Note that of the list, we should only need the Roslyn one. The others are dependencies used to run tests/build a test environment that shouldn't be relevant in sourcebuild.